src/components/Search.js

```diff
@@ -1,22 +1,16 @@
-import React, {Component} from 'react';
+import React from 'react';
```

Comment:
Created a Functional Stateless Component.  When you are only rendering props, use these.  Class extends component is used when you need to set state.